components/dashboard/package.json

```diff
@@ -36,7 +36,7 @@
     "@types/ua-parser-js": "^0.7.33",
     "@types/uuid": "^3.1.0",
     "babel-loader": "^8.0.6",
-    "css-loader": "^4.2.2",
+    "css-loader": "^0.28.11",
```

Comment:
I had to downgrade it as a follow-up of https://github.com/gitpod-io/gitpod/commit/95e75d59e38505d24245556aec8fa92f7fb186d9, otherwise css files were not properly bundled for the dashboard.